.github/workflows/universalnrp-test.yml

```diff
@@ -7,9 +7,27 @@ on:
     - cron: '0 20 * * *' # Every day at 12pm PST (UTC-8)
 
 jobs:
+  package:
+    name: Package
+    uses: ./.github/workflows/package-build.yml
+    strategy:
+      matrix:
+        target:
+          [
+            { os: ubuntu, version: 14.04, arch: amd64, dist: trusty, package-type: DEB },
+          ]
+    with:
+      target: ${{ matrix.target.os }}-${{ matrix.target.version }}
+      arch: ${{ matrix.target.arch }}
+      artifact: nrp-test-package
+      package-type: ${{ matrix.target.package-type }}
+      machine-config: true
+      release: ${{ github.event_name == 'pull_request' && false || true }}
```

Comment:
Move out this to original nrp-build.yml as that one is also used to build the package as artifacts that get posted in GitHub to be either published with MC later or posted under one of our releases.